main_files/main.py

```diff
@@ -200,7 +203,15 @@ def run_mcor(input_dict):
     # input_dict["existing_components_inputs"]["pv"] = PV(existing=True, pv_capacity=100, tilt=tilt, azimuth=azimuth,
     #         module_capacity=0.360, module_area=3, spacing_buffer=2,
     #         pv_tracking='fixed', pv_racking='ground')
-    # input_dict["existing_components_inputs"]["existing_components"] = {'pv': pv}
+    # input_dict["existing_components_inputs"]["existing_components"] = {"pv": ["pv"]}
+
+    input_dict["existing_components_inputs"]["existing_generator"] = Generator(existing=True, rated_power=500, num_units=1,
```

Comment:
I think we'll want to specify all of the parameters that are passed into the Generator object in the input_dict and then create the Generator object separately, probably in microgrid_optimizer.define_grid(). The code that will be creating the input_dict in gridpiq will not have access to the Generator class, only the run_mcor function. 

We should also do the same for any existing PV or Battery objects.